lib/include/jxl/decode.h

```diff
@@ -238,6 +243,39 @@ typedef enum {
    * image and always before JXL_DEC_FULL_IMAGE.
    */
   JXL_DEC_JPEG_RECONSTRUCTION = 0x2000,
+
+  /** Informative event by JxlDecoderProcessInput: The header of a box of the
+   * BMFF container format is decoded. The following API functions related to
+   * boxes can be used after this event:
+   * @see JxlDecoderSetBoxBuffer and JxlDecoderReleaseBoxBuffer: set and release
+   * a buffer to get the uncompressed box data.
+   * @see JxlDecoderGetBoxType get the 4-character uncompressed box typename.
+   * @see JxlDecoderBoxTypeEquals to compare with the 4-character uncompressed
+   * box typename.
+   * @see JxlDecoderIsBoxCompressed to query whether the box was stored as a
+   * compressed box of type "brob", or directly uncompressed.
+   * @see JxlDecoderGetBoxTypeRaw get the 4-character raw, "brob" if compressed,
+   * box typename.
+   * @see JxlDecoderBoxTypeRawEquals to compare with the raw box typename.
+   * @see JxlDecoderGetBoxSizeRaw to get the raw compressed box size.
+   * @see JxlDecoderSetBoxModeRaw to configure whether to get the box
+   * data uncompressed, or possibly compressed. Getting the raw box contents
+   * returns the same data and type unless JxlDecoderIsBoxCompressed is true. In
+   * the compressed box case, the raw typename is "brob" but the underlying
+   * typename can still be requested, and the data is brotli compressed. To get
+   * metadata, use the non-raw box mode. To perform more primitive operations
+   * with boxes, use the raw box mode and the non-raw type if needed. The buffer
```

Comment:
I think that the part starting with "The buffer" should be on a separate paragraph, no?